cmd/adapter/adapter.go

```diff
@@ -144,6 +161,9 @@ func (cmd *PrometheusAdapter) addFlags() {
 		"interval at which to re-list the set of all available metrics from Prometheus")
 	cmd.Flags().DurationVar(&cmd.MetricsMaxAge, "metrics-max-age", cmd.MetricsMaxAge, ""+
 		"period for which to query the set of available metrics from Prometheus")
+	cmd.Flags().StringVar(&cmd.MetricsPort, "metrics-port", cmd.MetricsPort, "Port for metrics")
+	cmd.Flags().StringVar(&cmd.MetricsHost, "metrics-host", cmd.MetricsHost, "Host for exposing metrics")
+	cmd.Flags().StringVar(&cmd.TLSConfig, "tls-config", cmd.TLSConfig, "TLS configuration for metrics server")
```

Comment:
Could please mention that the tls config is optional as it is done for the other TLS related flags.

Reply:
Done